docs/SetupAzureDatabase-1.md

```diff
@@ -0,0 +1,58 @@
+# Set up a database
+
+This guide explains how to set up a database for Scalar DL in Azure and how to configure helm charts for Scalar DL deployment.
+
+## Cosmos DB
+
+In this section, you will set up a Cosmos DB for Scalar DL.
+
+### Requirements
+
+* You must create a Cosmos DB account with `Core(SQL)` API.
+* You must change Cosmos DB `Default consistency` to `Strong`.
+* You must use Capacity mode as `Provisioned throughput`.
```

Comment:
Why it's a must?

Reply:
To deploy schema loader to Cosmos DB the `Capacity mode` should be `Provisioned throughput`, the deployment was failed with `Capacity mode` as `Serverless`